CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Version History
     * All Version bumps are required to update this file as well!!
 ----
 
+* 19.0.0 Removed job search version(s) prior to version 3.
```

Comment:
Not sure if you saw my other comment, but you should detail what the breaking change is here for those that consume this gem to better understand what changes they need to make.